loqusdb/build_models/variant.py

```diff
@@ -24,8 +24,10 @@ def check_par(chrom, pos):
             par(bool)
     """
     par = False
-
-    for interval in PAR.get(chrom,[]):
+    if genome_build is None:
+        genome_build=GRCH37
+    for interval in PAR[genome_build].get(chrom, []):
+        print(interval)
```

Comment:
```suggestion

```
:)

Reply:
Oops! Will remove!
Thanks for review. We don't need to merge quite yet. I haven't had time to test this on stage, and I will need to wait until hasta on Monday!